agents/python/opamp/http_client.py

```diff
@@ -35,7 +36,14 @@ def __init__(self, event, condition: threading.Condition):
         self.instance_uid = uuid7().__str__()
         self.remote_config_status = None
 
-    def start(self):
+
+    def start(self, python_version_supported: bool = None):
+        if not python_version_supported:
+            opamp_logger.warning("Python version not supported, sending disconnect message to OpAMP server...")
```

Comment:
might be useful to add the python version that we found and what is the minimum we support.

perhaps reuse the `last_error` below?